src/Umbraco.Web/Models/Mapping/ContentTypeMapDefinition.cs

```diff
@@ -224,11 +224,9 @@ private static void Map(PropertyTypeBasic source, PropertyType target, MapperCon
             target.Mandatory = source.Validation.Mandatory;
             target.ValidationRegExp = source.Validation.Pattern;
             target.Variations = source.AllowCultureVariant
-                // Set culture flag
-                ? target.Variations | ContentVariation.Culture
-                // Unset culture flag
-                : target.Variations & ~ContentVariation.Culture;
-
+                ? target.Variations.SetFlag(ContentVariation.Culture)
+                : target.Variations.UnsetFlag(ContentVariation.Culture);
+            
```

Comment:
I added the extension methods `SetFlag` and `UnsetFlag` to make it more readable, and also added unit tests for those extensions. Hope that's OK.